src/main/webapp/app/exercises/programming/shared/utils/programming-exercise-simulation-utils.ts

```diff
@@ -0,0 +1,13 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({ providedIn: 'root' })
+export class ProgrammingExerciseSimulationUtils {
+    /**
+     * Checks if the url includes the string "nolocalsetup', which is an indication
+     * that the particular programming exercise has no local setup
+     * @param urlToCheck the url which will be check if it contains the substring
+     */
+    hasNoLocalSetup(urlToCheck: string): boolean {
```

Comment:
I don't like the term "nolocalsetup"

Reply:
i renamed the method the new name is noVersionControlAndContinuousIntegrationAvailableCheck